internal/services/sentinel/sentinel_alert_rule_fusion_resource.go

```diff
@@ -137,19 +136,8 @@ func resourceSentinelAlertRuleFusionCreateUpdate(d *pluginsdk.ResourceData, meta
 	}
 	id := alertrules.NewAlertRuleID(workspaceID.SubscriptionId, workspaceID.ResourceGroupName, workspaceID.WorkspaceName, name)
 
-	if d.IsNewResource() {
```

Comment:
since we are here fixing this could we please split create and update?

Reply:
updated